### PR TITLE
CI fix for python 2 and numpy 1.14

### PR DESCRIPTION
**Pull request by nabobalis**
FOR NOW.

### CODE REVIEW

**Comment by nabobalis** [comment]
SQUASH MERGE

**Comment by Cadair** [comment]
but sunpy's commit history is littered with horrific strings of commits which are me fixing CI services :D well before GH had squash merge and before I knew what that was 😆

**Comment by nabobalis** [comment]
Well you'd be pleased to know the 0.8 branch has that since I used that to check the fix about 1 million times.

**Comment by Cadair** [comment]
👍

File: appveyor.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ environment:
       CONDA_CHANNELS: "conda-forge astropy-ci-extras"
       CONDA_DEPENDENCIES: "scipy matplotlib mock pandas requests beautifulsoup4 sqlalchemy scikit-image pytest-mock wcsaxes suds-jurko glymur"
       PIP_DEPENDENCIES: "hypothesis sunpy-sphinx-theme pytest-astropy"
-      NUMPY_VERSION: "stable"
+      NUMPY_VERSION: "1.13"
```
Comment (Cadair) [inline appveyor.yml]: 😆 Windows problems.

Comment (nabobalis) [reply appveyor.yml]: Oh yeah, I did not want to powershell some variables. 

I should though.

Comment (Cadair) [reply appveyor.yml]: I wouldn't worry

Comment (nabobalis) [reply appveyor.yml]: Well I was going to until you pointed it out.

Hurry up NumPy 1.14.1

**Comment by Cadair** [comment]
Travis fail is doctests which are being fixed in #2394

**Comment by nabobalis** [comment]
Yeah, the Py2 tests pass now.

**Pull request merged**